packages/tools/kolibri/package.json

```
@@ -0,0 +1,54 @@
+{
+	"name": "@public-ui/kolibri",
```

```suggestion
	"name": "@public-ui/kolibri-cli",
```
Vorschlag zur besseren Abgrenzung von `@public-ui/components`